Mini Challenge 2 Team 16/Mini Challenge 2 Team 16/Journal/JournalViewController.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  JournalViewController.swift
+//  Mini Challenge 2 Team 16
+//
+//  Created by Vincensa Regina on 20/06/22.
+//
+
+import UIKit
+
+class JournalViewController: UIViewController {
+    
+    @IBOutlet weak var btnGoal: UIButton!
+    @IBOutlet weak var txtCurrBalance: UILabel!
+    @IBOutlet weak var pbGoal: UIProgressView!
+    @IBOutlet weak var txtGoalAmount: UILabel!
+    @IBOutlet weak var txtDate: UILabel!
+    @IBOutlet weak var scSection: UISegmentedControl!
+    @IBOutlet weak var btnLeft: UIButton!
+    @IBOutlet weak var btnRight: UIButton!
+    @IBOutlet weak var tbJournal: UITableView!
+    @IBOutlet weak var txtPercent: UILabel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Table View Setting
+        tbJournal.delegate = self
+        tbJournal.dataSource = self
+        self.registerTableView()
+    }
+    
+    @IBAction func valueChanged(_ sender: Any) {
+        switch scSection.selectedSegmentIndex
+            {
+            case 0:
+               break
+            case 1:
+                break
+            default:
+                break
+        }
+    }
+    
+    private func registerTableView() {
+        // Table View Cells
+        let textFieldCell = UINib(nibName: "JournalTableViewCell", bundle: nil)
+        self.tbJournal.register(textFieldCell, forCellReuseIdentifier: "JournalCustomCell")
+        
+        // Table Header
+        tbJournal.register(JournalHeaderView.self,
+                           forHeaderFooterViewReuseIdentifier: "journalHeader")
+        
+        //Section Header Height
```

Comment:
when your code should have a comments. so it means that your code is not readable enough. variable naming, function naming should be very2 clear.